Add Code of Conduct

arshiamufti opened the pull request:
In case you missed it on slack--we're adding a formspree to the contact info and then later a public key that people can use to encrypt their complaints.

annalorimer commented:
Thought: Do we want to give a guideline for writing complaints? i.e what information to include (if possible) such as date, location etc.

arshia-stripe commented:
I think it's pretty obvious what should be put down in a complaint. Happy to add it if you think it's necessary but imo it's fine as it is.

annalorimer commented:
@arshiamufti I'm not sure it's so obvious, especially if the person making the complain is upset. Either way it won't hurt.

evykassirer commented on _pages/coc.md:
```diff
@@ -7,7 +7,82 @@ permalink: /coc/
 
-Coming soon! We're just polishing it up.
+Our Code of Conduct is our commitment to the community to provide a welcoming,
+safe space for all its participants. All speakers, sponsors and volunteers at
```
oxford comma pls

evykassirer commented on _pages/coc.md:
```diff
@@ -7,7 +7,82 @@ permalink: /coc/
+
+**Unacceptable behaviors include:** intimidating, harassing, abusive,
+discriminatory, derogatory or demeaning speech or actions by any participant in
+our community online, at all related events and in one-on-one communications
```
oxford comma

evykassirer commented on _pages/coc.md:
```diff
@@ -7,7 +7,82 @@ permalink: /coc/
+## Experiencing Unacceptable Behavior ##
+
+If you are subject to or witness unacceptable behavior, or have any other
+concerns, please [contact us](#contact-information).
```
you can put our google group email

evykassirer commented on _pages/coc.md:
```diff
@@ -7,7 +7,82 @@ permalink: /coc/
+## Addressing Grievances ##
+
+If you feel you have been falsely or unfairly accused of violating this Code of
+Conduct, [contact us](#contact-information) with a concise description of your
```
this link works yeah?

evykassirer replied:
Why "concise"? I feel like we should just let them write whatever they need to

arshiamufti replied:
yes this link works. I'll change it to "clear"

arshiamufti replied:
actually, let's delete this altogether

evykassirer replied:
I kinda like the explicit "if something bad happens you can email us"

without this section there's no "so what do I do"

arshiamufti commented on _pages/coc.md:
```diff
@@ -7,7 +7,82 @@ permalink: /coc/
+
+## Contact Information ##
+
+Email: [starconuw@googlegroups.com](mailto:starconuw@googlegroups.com)
```
should i change this to our gmail?

arshiamufti replied:
cc @evykassirer

evykassirer replied:
that depends. is there any preference which is our public facing email?

we could just cc the gmail on everything and it'd forward to us through the google group, but I think I could also just set up the gmail to forward to all of us and keep the emails in there instead of the google group?

this is so confusing

arshiamufti replied:
yeah this is confusing. Okay let's use uwstarcon@gmail.com just for mailchimp. And the googlegroups email as public facing. For personal interactions w sponsors and speakers, we'll just use our personal emails (just like we did for wics)

arshiamufti commented:
addressed feedback and added formspree. However, we still have to setup our account which can only be done when this is deployed to github pages ([step 3 here](https://formspree.io/)) so this will have to be merged into master for us to complete that step.

evykassirer commented:
Looks good to me!

Can you open an issue for setting up formspree for the weekend?
